miasm2/jitter/bn.h

```diff
@@ -0,0 +1,141 @@
+#ifndef __BIGNUM_H__
```

Comment:
It would be nice to add the source of the original library

Reply:
Done